src/test/regress/expected/distributed_types_0.out

```diff
@@ -100,6 +100,20 @@ SELECT * FROM t4;
 
 -- ALTER TYPE ... ADD VALUE does not work in transactions
 COMMIT;
+-- verify order of enum labels
+SELECT string_agg(enumlabel, ',' ORDER BY enumsortorder ASC) FROM pg_enum WHERE enumtypid = 'type_tests.te2'::regtype;
+ string_agg 
+------------
+ yes,no
```

Comment:
is it worth making this list longer? or is the order deterministic?

Reply:
I think it is deterministic, I scan the index that contains the order column as the second column. But am not super familiar with how to scan indices. Didn't see a ASC/DESC option for a ScanKey.